accounts/login_failure.php

```diff
@@ -26,6 +26,8 @@
     'no_username'      => _("You did not supply a username."),
     'no_password'      => _("You did not supply a password."),
     'auth_failure'     => _("Unable to authenticate. The username/password may be incorrect or your account may be locked."),
+    'unknown_failure'  => sprintf(_("An unexpected failure occurred, please contact the <a href='%s'>site manager</a>."), "mailto:$site_manager_email_addr"),
```

Comment:
I'd probably say `a` site manager instead of `the` site manager

Reply:
I'm fine either way, this is using the same wording as the other paragraph at the bottom of the file.

Reply:
Just grammatical pickiness -- `the` is specific, and implies that there's only one.